### DISCUSSION TITLE
Where are templatefields in Bolt5?

### DISCUSSION THREAD

**Discussion by PulsarFX** [bolt/core#3409] [Q&A]
I'm currently trying to convert a Bolt 3 website to Bolt 5.
In the old one I used lots of templatefields. ( https://docs.boltcms.io/3.7/templating/template-specific-fields ) 
Is this feature still present in Bolt5? I couldn't find any information about it in the bold5 docs. 
If it has been dropped, what is an alternative approach to mimic the old behavior?

**Comment by nickWardDesigns** [comment]
I had the same question, and ended up emailing support directly. Templatefields were removed due to confusion, so now the best approach is to use [collections](https://docs.boltcms.io/5.0/fields/collection) in your contenttypes to add specific fields you may need.
